data/WI/incentives.json

```diff
@@ -0,0 +1,410 @@
+[
+  {
+    "id": "WI-1",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "wi_focusOnEnergy_insulation&AirSealingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1125,
+      "minimum": 675,
+      "maximum": 1125
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $1,125 for ENERGY STAR Qualified Air Sealing. Must be completed with Trade Ally contractor installed attic or wall insulation improvements."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
```

Comment:
For these first 3, it seems like there are two versions of each incentive – one income qualified and one not. I would say the right thing to do is to split those rows (in the spreadsheet), specify the more exact amounts, and update the descriptions to indicate whether they are low-income qualified or not. Then only one of each pair will get the low-income tag.

You might need to redo the IDs as a result, but that's fine, they aren't checked-in yet anyway.